academic-services/graph/samples-databricks-network-similarity.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Network Similarity Sample (PySpark)
+description: Network Similarity Sample (PySpark)
+services: microsoft-academic-services
+ms.topic: extra
+ms.service: microsoft-academic-services
+ms.date: 9/23/2020
+---
+# Network Similarity Sample (PySpark)
+
+In this sample, you compute network similarity score and top related entities using Azure Databricks.
```

Comment:
In this sample, you compute the network similarity score and top related entities using Azure Databricks.